src/app/components/confirmBtcTransaction/transactionSummary.tsx

```diff
@@ -85,6 +97,11 @@ function TransactionSummary({
   const feesHaveInscribedRareSats = feeOutput?.inscriptions.length || feeOutput?.satributes.length;
   const showInscribeRareSatWarning = paymentHasInscribedRareSats || feesHaveInscribedRareSats;
 
+  const satsToFiat = (sats: string) =>
+    getBtcFiatEquivalent(new BigNumber(sats), BigNumber(btcFiatRate)).toNumber().toFixed(2);
```

Comment:
```suggestion
    getBtcFiatEquivalent(new BigNumber(sats), new BigNumber(btcFiatRate)).toNumber().toFixed(2);
```

Reply:
Does `new` actually do anything here? Or are we enforcing it for consistency reasons?

Reply:
Nope, not needed. But it's good to be consistent 😋 Thanks for flagging

Reply:
it was for consistency, as I don't think it matters with BigNumber, but here is what I found out of curiosity: 
![image](https://github.com/secretkeylabs/xverse-web-extension/assets/6109710/e671687d-d1e9-41b6-8700-d5d10e027e49)